src/commands/deploy.ts

```diff
@@ -59,14 +60,18 @@ export async function deploy(ui: IAzureUserInput, telemetryProperties: Telemetry
                 }
             }
         }
-
         const client: SiteClient = node.treeItem.client;
-
         const language: ProjectLanguage = await getProjectLanguage(deployFsPath, ui);
         telemetryProperties.projectLanguage = language;
         const runtime: ProjectRuntime = await getProjectRuntime(language, deployFsPath, ui);
         telemetryProperties.projectRuntime = runtime;
 
+        if (language === ProjectLanguage.Python) {
+            await runFromZipDeploy(ui, tree, node, deployFsPath, outputChannel);
+            outputChannel.appendLine(localize('deployComplete', '>>>>>> Deployment to "{0}" completed. <<<<<<', client.fullName));
+            return;
```

Comment:
Are you referring to this? https://github.com/Microsoft/vscode-azuretools/blob/626070c466529f184b438bd6e825971b5ccf9a12/appservice/src/deploy/deploy.ts#L65-L70

I thought the url only got displayed if it was a Web App since it doesn't make a lot of sense for a user to hit the Function app's url.

Reply:
Nope not the _function app_ url (which you are correct is not displayed), but the _function_ urls.

It's in this file. See https://github.com/Microsoft/vscode-azurefunctions/blob/master/src/commands/deploy.ts#L115